packages/react-components/src/components/Tooltip/Tooltip.stories.tsx

```diff
@@ -0,0 +1,422 @@
+import * as React from 'react';
+import { ComponentMeta } from '@storybook/react';
+import { Button } from '../Button';
+import { ITooltipProps, Tooltip as TooltipComponent } from './Tooltip';
+import {
+  DropDown,
+  Smiles,
+  CannedResponse,
+  Attachment,
+} from '@livechat/design-system-icons/react/material';
+
+import { Icon } from '../Icon';
+
+import './Tooltip.stories.css';
+import { Simple } from './Simple';
+import { Info as TooltipInfoComponent } from './Info';
+import { Interactive as TooltipInteractiveComponent } from './Interactive';
+import { UserGuide as TooltipUserGuideComponent } from './UserGuide';
+import { UserGuideStep } from './UserGuideStep';
+import beutifulImage from './placeholder.png';
+
+const tooltipPlacements = [
+  'bottom',
+  'bottom-end',
+  'bottom-start',
+  'left',
+  'left-end',
+  'left-start',
+  'right',
+  'right-end',
+  'right-start',
+  'top',
+  'top-end',
+  'top-start',
+];
+
+const isVisibleOptions = [true, false, undefined];
+
+const tooltipThemes = ['invert', 'important', 'default'];
+
+export const Tooltip = (args: any): React.ReactElement => (
+  <div
+    style={{
+      height: '100vh',
+      display: 'flex',
+      justifyContent: 'center',
+      alignItems: 'center',
+    }}
+  >
+    <span className={'tooltip-icon-wrap'}>
+      <TooltipSimpleExample {...args}></TooltipSimpleExample>
+    </span>
+    <span className={'tooltip-icon-wrap'}>
+      <Icon source={CannedResponse}></Icon>
+    </span>
+    <span className={'tooltip-icon-wrap'}>
+      <Icon source={Attachment}></Icon>
+    </span>
+  </div>
+);
+
+Tooltip.args = {
+  placement: 'bottom',
+  isVisible: true,
```

Comment:
IMO for a basic tooltip example, it would be great to showcase its default behavior with a hover state. That way we shouldn't provide the `isVisisble` value.

```suggestion
```